website/docs/docs/cloud/migration.md

```diff
@@ -53,7 +53,7 @@ Complete all of these items to ensure your <Constant name="cloud" /> resources a
 
 Use one of these two URL login options:
 
-- `us1.dbt.com.` If you were previously logging in at `cloud.getdbt.com`, you should instead plan to log in at us1.dbt.com. The original URL will still work, but you’ll have to click through to be redirected upon login.
+- `us1.dbt.com.` If you were previously logging in with a username and password at `cloud.getdbt.com`, you should instead plan to log in at us1.dbt.com. The original URL will still work, but you’ll have to click through to be redirected upon login. If you have single sign-on configured, you will use the unique URL listed in the SSO account settings. 
```

Comment:
```suggestion
- `us1.dbt.com.` If you were previously logging in with a username and password at `cloud.getdbt.com`, you should instead plan to log in at us1.dbt.com. The original URL will still work, but you’ll have to click through to be redirected upon login. If you have single sign-on configured, you will use the unique URL listed in the SSO account settings (ex: `ab123.us1.dbt.com`). 
```

Reply:
The SSO URL changes at the end doesn't it @ryancharris - so it would be - `us1.dbt.com/enterprise-login/login-slug` Does it still get the prefix?